nikola/data/themes/base/templates/listing.tmpl

```diff
@@ -6,10 +6,10 @@ ${ui.bar(crumbs)}
 %if folders or files:
 <ul>
 % for name in folders:
-    <li><a href="${name|u}"><i class="icon-folder-open"></i> ${name|h}</a>
+    <li><a href="${name|u}">${name|h}</a>
```

Comment:
To make styling possible, please add classes to links, eg. "listing-folder" and "listing-file".